scripts/portconfig

```diff
@@ -280,6 +292,10 @@ def main():
                         help = 'port advertised interface types', default=None)
     parser.add_argument('-lt', '--link-training', type = str, required = False,
                         help = 'port link training mode', default=None)
+    parser.add_argument('-P', '--tx-power', type=float, required=False,
+                                            help='Tx output power(dBm)', default=None)
+    parser.add_argument('-F', '--laser-freq', type=int, required=False,
```

Comment:
>int [](http://example.com/codeflow?start=51&length=3)

Check zero and negative? Also in yang model.

Reply:
the actual frequency range supported depends on the ZR module plugged, so module will reject 0 and -ve frequency anyways.

Reply:
@qiluo-msft added check now